### PR TITLE
Us032   update readme

### PR DESCRIPTION
**Pull request by bannister-samuel**

### CODE REVIEW

**Review by cdrainkainos — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1 +1,34 @@
...
\ No newline at end of file
+# TeamC-API
+
+Database
+---
+1. The api connects to database: AgileSprints_SamB
+
+Config
+---
+1. Create an empty file called employeedb.properties in <directory_clone_was_ran>/TeamC-API/
```
Comment (boydstepnew) [inline README.md]: Can just call this the project root instead of the path

File: README.md [hunk 2]
```diff
@@ -1 +1,34 @@
...
+2. The following environment variables need to be set to enable database connection:
+- host
+- user
+- password
+- db
+
+How to start the java-dropwizard-swagger-jdbc application
```
Comment (boydstepnew) [inline README.md:17]: Can you bold these headings?

File: README.md [hunk 3]
```diff
@@ -1 +1,34 @@
-# TeamC-API
\ No newline at end of file
+# TeamC-API
+
+Database
```
Comment (boydstepnew) [inline README.md:4]: Perhaps mention Java 11?

File: README.md (hunk 1 above)
Comment (bannister-samuel) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (bannister-samuel) [reply README.md:17]: done

File: README.md (hunk 3 above)
Comment (bannister-samuel) [reply README.md:4]: moved from after DB instructions

**Review by boydstepnew — APPROVED** [review]

**Pull request merged**